CHANGELOG.md

```diff
@@ -42,6 +42,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - [#206 clearing Setting's cache is now done via DoctrineEventListener](https://github.com/shopsys/shopsys/pull/206)
     - `EntityManagerFacade` was removed
     - Doctrine identity map can be cleared via `EntityManager::clear()` directly
+- [#244 Redesign of administration](https://github.com/shopsys/shopsys/pull/244)
+    - full-width layout
+    - colors changed to match new Shopsys CI
+    - main menu moved to the left panel along with settings menu
+    - menu items do not have type anymore, which lead to simplification of the code
+    - menu was restructured
```

Comment:
third level is disabled?

Reply:
Commit 94a1384601656617b89f1208c9aceafc820652f9 dropped. I've slightly reordered some items after a consultation with @LukasHeinz in d85ff8779.